manticore/platforms/evm.py

```diff
@@ -94,10 +94,11 @@ def dump(self, stream, state, mevm, conc_tx=None):
 
         :param stream: Output stream to write to. Typically a file.
         :param manticore.core.state.State state: state that the tx exists in
-        :param manticore.ethereum.ManticoreEVM mevm: manticore instance
+        :param ethereum.manticore.ManticoreEVM mevm: manticore instance
```

Comment:
should this be `manticore.ethereum.manticore.ManticoreEVM`?

Reply:
Either `manticore.ethereum.ManticoreEVM` or as you wrote, will use the first one.